plugins/woocommerce-blocks/tests/e2e/tests/cart/cart-block.shopper.block_theme.side_effects.spec.ts

```diff
@@ -128,4 +131,151 @@ test.describe( 'Shopper → Cart block', () => {
+		await page
+			.getByLabel( `Remove ${ SIMPLE_PHYSICAL_PRODUCT_NAME } from cart` )
+			.click();
+
+		// Verify product is removed from the cart'
+		await expect(
+			page.getByRole( 'heading', {
+				name: 'Your cart is currently empty!',
+			} )
+		).toBeVisible();
+	} );
+
+	test( 'User can update product quantity', async ( {
+		frontendUtils,
+		page,
+	} ) => {
+		await frontendUtils.goToShop();
+		await frontendUtils.addToCart( SIMPLE_VIRTUAL_PRODUCT_NAME );
+		await frontendUtils.goToCart();
+
+		// Via the input field
+		await page
+			.getByLabel(
+				`Quantity of ${ SIMPLE_VIRTUAL_PRODUCT_NAME } in your cart.`
+			)
+			.fill( '4' );
+
+		// Verify the "Proceed to Checkout" button is disabled during network request
+		await expect(
+			page.getByRole( 'button', { name: 'Proceed to Checkout' } )
+		).toBeDisabled();
+
+		// Verify the "Proceed to Checkout" button is enabled after network request
+		await expect(
+			page.getByRole( 'link', { name: 'Proceed to Checkout' } )
+		).toBeEnabled();
+
+		await expect(
+			page.getByLabel(
+				`Quantity of ${ SIMPLE_VIRTUAL_PRODUCT_NAME } in your cart.`
+			)
+		).toHaveValue( '4' );
+
+		// Via the plus button
+		await page
+			.getByLabel(
+				`Increase quantity of ${ SIMPLE_VIRTUAL_PRODUCT_NAME }`
+			)
+			.click();
+		// Verify the "Proceed to Checkout" button is disabled during network request
+		await expect(
+			page.getByRole( 'button', { name: 'Proceed to Checkout' } )
+		).toBeDisabled();
+
+		// Verify the "Proceed to Checkout" button is enabled after network request
+		await expect(
+			page.getByRole( 'link', { name: 'Proceed to Checkout' } )
+		).toBeEnabled();
+
+		await expect(
+			page.getByLabel(
+				`Quantity of ${ SIMPLE_VIRTUAL_PRODUCT_NAME } in your cart.`
+			)
+		).toHaveValue( '4' );
+
+		// Via the minus button
+		await page
+			.getByLabel( `Reduce quantity of ${ SIMPLE_VIRTUAL_PRODUCT_NAME }` )
+			.click();
+		// Verify the "Proceed to Checkout" button is disabled during network request
+		await expect(
+			page.getByRole( 'button', { name: 'Proceed to Checkout' } )
+		).toBeDisabled();
+
+		// Verify the "Proceed to Checkout" button is enabled after network request
+		await expect(
+			page.getByRole( 'link', { name: 'Proceed to Checkout' } )
+		).toBeEnabled();
+
+		await expect(
+			page.getByLabel(
+				`Quantity of ${ SIMPLE_VIRTUAL_PRODUCT_NAME } in your cart.`
+			)
+		).toHaveValue( '4' );
+	} );
+
+	test( 'User can proceed to checkout', async ( { frontendUtils, page } ) => {
+		await frontendUtils.goToShop();
+		await frontendUtils.addToCart( SIMPLE_VIRTUAL_PRODUCT_NAME );
+		await frontendUtils.goToCart();
+
+		// Click on "Proceed to Checkout" button
+		await page.getByRole( 'link', { name: 'Proceed to Checkout' } ).click();
+
+		// Verify that you see the Checkout Block page
+		await expect(
+			page.getByRole( 'heading', { name: 'Checkout' } )
+		).toBeVisible();
+	} );
+
+	test( 'User can see Cross-Sells products block', async ( {
```

Comment:
I think we need to enable cross-sells products first. This test is failing for me and I also can't see any cross-sells product in chromium: 

![image](https://github.com/woocommerce/woocommerce/assets/11503784/15c29fbb-f963-44d6-9086-2341d005c596)

Reply:
Before the monorepo merge, we were using a helper plugin for setting up the Cross-Sells products. The corresponding part can be found on https://github.com/woocommerce/woocommerce-blocks/blob/4cdab7effbee82dad494a188a7026d4d8dab82a4/tests/mocks/woo-test-helper/woo-test-helper.php#L98C1-L127

I'll investigate if we can somehow reuse that existing functionality.